PSKNodeServer/PSKNodeServer/API/scanditScan/ScanditScannerViewController.swift

```diff
@@ -6,22 +6,78 @@
 //
 
 import UIKit
+import ScanditBarcodeCapture
 
 class ScanditScannerViewController: UIViewController {
     @IBOutlet weak var apiKeyLabel: UILabel!
     private let scanditApiKey: String
+    private let symbologies: [Symbology]
+    private let dataCaptureContext: DataCaptureContext
+    private var barcodeCapture: BarcodeCapture
     
     override func viewDidLoad() {
         super.viewDidLoad()
         apiKeyLabel.text = scanditApiKey
     }
     
+    override func viewWillAppear(_ animated: Bool) {
+        barcodeCapture.addListener(self)
```

Comment:
Not sure how is it implemented in Scandit. But usually it's good practice also to remove listener when it's not needed (e.g. viewWillDissappear)